### PR TITLE
[Improvement](Storage) Lazy init mini_download dir

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

image
Mini_ Download is only used by streamload to import some special formats, such as parquet. If these special formats are not imported, it will result in invalid warning logs.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run build

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by kaijchen — APPROVED** [review]

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

File: be/src/runtime/load_path_mgr.cpp [hunk 1]
```diff
@@ -91,8 +88,8 @@ Status LoadPathMgr::init() {
 
 Status LoadPathMgr::allocate_dir(const std::string& db, const std::string& label,
                                  std::string* prefix) {
-    if (_path_vec.empty()) {
-        return Status::InternalError("No load path configured.");
+    for (auto& store_path : _exec_env->store_paths()) {
+        _path_vec.push_back(store_path.path + "/" + MINI_PREFIX);
     }
```
Comment (dataroaring) [inline be/src/runtime/load_path_mgr.cpp:93]: The size of _path_vec increases multi times.

**Review by dataroaring — CHANGES_REQUESTED** [review]

File: be/src/runtime/load_path_mgr.cpp (hunk 1 above)
Comment (HHoflittlefish777) [reply be/src/runtime/load_path_mgr.cpp]: done

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**